config/arch/sh.in.2

```diff
@@ -4,10 +4,30 @@ choice
     bool
     prompt "Variant"
 
+config ARCH_SH_SH1
+    bool
+    prompt "sh1"
+
+config ARCH_SH_SH2
```

Comment:
Curious which libc variant does your configuration use? Baremetal?
I am asking because this change may affect the multilib PR I posted - gcc for SuperH allows to select the set of MCUs for the set of multilibs to be built - and that requires some extra steps for uClibc (see https://github.com/stilor/crosstool-ng/blob/e42e353a64b518895bf9909b9f966544ead5f255/scripts/build/arch/sh.sh for example).
If using baremetal, probably a good idea would be to save your configuration as a sample - we don't have sh*-*-none samples.